deployments/Dockerfiles/autonomy/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG AEA_VERSION=latest
+
+FROM valory/open-aea-user:${AEA_VERSION}
+
+ARG AUTHOR=default_author
+ARG AEA_AGENT
+
+COPY openssl.cnf /etc/ssl
+COPY requirements.txt /home/ubuntu
+
+WORKDIR /home/ubuntu
+ENV PATH=$PATH:/home/ubuntu/.local/bin
+
+RUN sudo apt-get update --fix-missing
+RUN sudo apt-get autoremove
+RUN sudo apt-get autoclean
+RUN sudo apt-get install python3.10 python3.10-dev git -y
+RUN cd /usr/bin && sudo rm python3 && sudo ln -s python3.10 python3 && sudo ln -s python3.10 python
+
+# used in Docker-compose to wait until Hardhat node is up
+RUN sudo apt-get install net-tools -y
+RUN python -m pip install --upgrade pip
+
+WORKDIR /home/ubuntu
+
+RUN pip3 install -r requirements.txt
+RUN aea init --reset --remote --ipfs --author ${AUTHOR}
```

Comment:
The thing is, the author here will be us, whereas it will be another one in the runtime image.... I think we basically should not init here...

Reply:
We can mount a config, then template that in the deployment chart.
This will enforce consistency.

Users would need to mount the init file however

Reply:
Ohk, I'll move this to runtime image, read the author name from local CLI config and pass it as the build arg.

Reply:
this actually makes loads of sense now i think about it, we could mount if from the user local aea config